### PR TITLE
feat(TDI-40427): Duplicate of tSnowflakeOutput

### PR DESCRIPTION
**Pull request by olhava**
* Changes for future overrides

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/42/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/42/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aovarnavina%2FTDI-40427_duplicate_tSnowflakeOutput)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/48/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/48/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aovarnavina%2FTDI-40427_duplicate_tSnowflakeOutput)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/54/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/54/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aovarnavina%2FTDI-40427_duplicate_tSnowflakeOutput)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/27/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/27/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aovarnavina%2FTDI-40427_duplicate_tSnowflakeOutput)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/37/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/37/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aovarnavina%2FTDI-40427_duplicate_tSnowflakeOutput)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/38/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/38/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aovarnavina%2FTDI-40427_duplicate_tSnowflakeOutput)

File: components/components-snowflake/components-snowflake-runtime/src/main/java/org/talend/components/snowflake/runtime/SnowflakeWriter.java [hunk 1]
```diff
@@ -249,4 +201,75 @@ public Schema getSchema() throws IOException {
         });
     }
 
+    protected TSnowflakeOutputProperties getProps() {
```
Comment (mbasiuk-talend) [inline components/components-snowflake/components-snowflake-runtime/src/main/java/org/talend/components/snowflake/runtime/SnowflakeWriter.java]: You already have this properties retrieved from sink. Don't need this method at all. If you want to get properties use sink.getSnowflakeOutputProperties()

**Pull request merged**